torch/distributed/rpc/api.py

```diff
@@ -78,6 +79,11 @@ def _init_rpc(
         raise RuntimeError("RPC is already initialized")
 
     # Initialize RPC.
+    if not isinstance(rpc_timeout, datetime.timedelta):
+        raise RuntimeError(
+            "`rpc_timeout` must be a `datetime.timedelta`."
```

Comment:
nit: I don't think we should have backticks (`) in exception messages. This would be displayed on the user's terminal and there is no markdown there.

Reply:
@pritamdamania87 

I was not expecting a markdown format. But want to make clear that it's a python variable. Any other suggestion to make it distinct? or you think plain text is fine?